### PR TITLE
Update kubernetes-executor.md

### PR DESCRIPTION
**Pull request by RNHTTR**
Add clarity on k8s executor resources

### CODE REVIEW

File: astro/kubernetes-executor.md [hunk 1]
```diff
@@ -11,6 +11,22 @@ On Astro, you can configure Kubernetes executor in the following ways:
 
 This document describes how to configure individual task Pods for different use cases. To configure defaults for all Kubernetes executor task Pods, see [Configure Kubernetes Pod resources](configure-deployment-resources.md#configure-kubernetes-pod-resources).
 
+## Configure resources for the Kubernetes executor
+### CPU and memory quotas
+Resource quotas define the maximum available resources for an Airflow Deployment. Resource quotas for the Kubernetes executor follow a 1:2 ratio of vCPUs to GiBs of memory. For example, if a deployment is configured to limit Airflow to 50 vCPUs, then Airflow will also be limited to 100 GiBs of memory and vice versa. The maximum quota is 400 vCPU / 800 GiB RAM.
```
Comment (melugoyal) [inline astro/kubernetes-executor.md:16]: `for an Airflow Deployment` might be a bit confusing, as the quota only applies to KE pods, not the scheduler, webserver, etc

File: astro/kubernetes-executor.md [hunk 2]
```diff
@@ -11,6 +11,22 @@ On Astro, you can configure Kubernetes executor in the following ways:
...
+
+### Max pod size
+The max pod size configures the maximum amount of resources that a given Airflow task can consume. Max pod size quotas for the Kubernetes executor follow a 1:2 ratio of vCPUs to GiBs of memory. For example, if the maximum memory is set to 20GiB, the number of vCPUs will be limited to 10 and vice versa. The maximum max pod size is currently 27 vCPU / 54 GiB RAM for Standard Clusters and 12 vCPU / 24 GiB RAM for Dedicated Clusters.
```
[the remark this answers is not in the document]
Comment (melugoyal) [reply astro/kubernetes-executor.md:19]: or i guess maybe just replace the word `configures` with `is` lol

**Review by melugoyal — COMMENTED** [review]
left a couple small comments, but overall looks good. i'm a bit surprised we never documented this stuff, are we sure it's not somewhere else in the docs? potentially under some KPO-specific docs or something? (since the quotas/max/default pod size stuff all works the same for KE and KPO)

**Comment by RNHTTR** [comment]
> left a couple small comments, but overall looks good. i'm a bit surprised we never documented this stuff, are we sure it's not somewhere else in the docs? potentially under some KPO-specific docs or something? (since the quotas/max/default pod size stuff all works the same for KE and KPO)

It is documented under [Configure a deployment](https://docs.astronomer.io/astro/configure-deployment-resources#configure-kubernetes-pod-resources), but I think it bears under the k8s executor section. I guess I should just link back to this section instead of repeating it...

**Pull request closed without merging**